content/en/platform/corda/1.6/cenm/backup-recommendations.md

```diff
@@ -0,0 +1,49 @@
+---
+aliases:
+- /cenm-backup-recommendations.html
+date: '2024-06-12'
+menu:
+  cenm-1-6:
+    identifier: cenm-1-6-cenm-backup-recommendations
+    parent: cenm-1-6-operations
+    weight: 185
+tags:
+- backup
+title: CENM backup recommendations
+---
+
+# CENM backup recommendations
+
+Various components of CENM read their configuration from the file system, and persist data to a database or into files on disk.
+Given that hardware can fail, operators of IT infrastructure must have a sound backup strategy in place. CENM holds important network information, such as approved and revoked network participants, the network map, and network parameters. While the latest network parameters are saved in a configuration file, the rest could not be recovered. As such, R3 strongly recommends implementing a comprehensive backup strategy.
```

Comment:
I think remove this:

~~While the latest network parameters are saved in a configuration file, the rest could not be recovered. As such,~~

Reply:
removed